### ISSUE TITLE
`<pf-switch>`: Missing required aria attribute

### ISSUE THREAD

**Issue by kylebuch8** [patternfly/patternfly-elements#2697]
## Description of the issue

The `pf-switch` is missing the required aria-checked attribute.
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/checkbox_role

### Impacted component(s)
- pf-switch

### Steps to reproduce
- Go to https://patternflyelements.org/components/switch/
- Observe that the aria-checked attribute it missing.

### Expected behavior

There should be an aria-checked attribute present indicating the state of the switch since the switch has a role of checkbox.

### Screenshots

![image](https://github.com/patternfly/patternfly-elements/assets/330256/2fadb74c-a45e-4451-a7ef-65c609436288)

**Comment by nikkimk** [comment]
@bennypowers element internals aren't showing up in the accessibility tree for `pf-switch` and `pf-jump-links`

**Comment by kylebuch8** [comment]
Should the role of pf-switch be `role="switch"`? (Thanks @bexter89)

**Comment by bennypowers** [comment]
@nikkimk either it's our fault and internals controller is broken, or it's the browser, or it's a polyfill bug. we should first see if there's a new version of the polyfill. then we should try it on various browsers with the polyfill NOT included and see if that helps.

worst case is that browsers aren't exposing this info to AT, in which case there's not much to be done about it other than throw up our hands and sprout attrs.

**Comment by bennypowers** [comment]
should be closed by #2702
